### ISSUE TITLE
Please add a requirements.txt file

### ISSUE THREAD

**Issue by lol768** [rougier/gl-agg#6]
For anyone else who ran into the same issues as me (symptoms: `RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility`, demos immediately dying when executed), these are the versions I installed with success:

```
sudo pip install  scipy==0.13.2
sudo pip install numpy==1.7
sudo pip install cython==0.20
```

**Comment by rougier** [comment]
Thanks for report. Do you know what does the error mean ? With laster numpy/scipy/cython version it does not work ?

**Comment by lol768** [comment]
Yeah, think so - I had to downgrade to those versions to run the demos.

It's not necessarily a problem, normally I'd make a `venv` and just install whatever dependency versions the project requires. In this case what was more difficult was guessing which versions the code would've been built against when you originally wrote it.

**Comment by rougier** [comment]
Sorry for the long delay. Could you make a PR with the proposed `requirements.txt` file?
